lib/src/dio-interceptor-wrapper.dart

```diff
@@ -52,7 +52,7 @@ class SuperTokensInterceptorWrapper extends Interceptor {
     }
 
     SuperTokensTokenTransferMethod tokenTransferMethod =
-        SuperTokens.config.tokenTransferMethod!;
+        SuperTokens.config.tokenTransferMethod;
```

Comment:
Null assertion operator is useless here, because tokenTransferMethod marked as late.

![image](https://github.com/supertokens/supertokens-flutter/assets/74528634/513c3e78-0e45-4ff8-acf5-04c3732e515b)
 
 And static type check shows that: "The '!' will have no effect because the receiver can't be null."